src/backend/Level0/CHIPBackendLevel0.hh

```diff
@@ -305,8 +305,10 @@ public:
     // go through all pools and try to get an allocated event
     for (size_t i = 0; i < EventPools_.size(); i++) {
       CHIPEventLevel0 *Event = EventPools_[i]->getEvent();
-      if (Event)
+      if (Event) {
+        Event->Deleted_ = false;
```

Comment:
This is redundant. The `Deleted_` flag is reset by the `getEvent()`call in the line 307 (LZEventPool::getEvent() -> CHIPEventLevel0::reset() -> CHIPEventLevel0::markDeleted(false)).

Reply:
wasn't working for Release, I believe. I was getting "event use after delete" errors. I need to clean up this branch, though.